### ISSUE TITLE
Highlight following the cursor

### ISSUE THREAD

**Issue by x0zzz** [t9md/atom-quick-highlight#22]
Is there a way to enable highlighting whatever is under the cursor without any need for shortcut to be pressed, i.e. wherever the cursor goes, the highlight changes/follows to the word under it?
Or is there maybe a package I could look into? I couldn't find any on my own...

I would really appreciate any help because I find such feature extremely useful.

Thanks for all the great work you do.

**Comment by t9md** [comment]
No, quick-highlight package currently have no such feature for now.
I quickly searched that package and I too can't find that package.

I'll quickly check if I can implement that feature in this package.

**Comment by x0zzz** [comment]
@t9md did you have time to check? Can it be done within quick-highlight? Or do I need to hack it or write something on my own?

**Comment by t9md** [comment]
I tried, and partially success, but was not easy.  So I postponed to polish further.
I haven't published my stopped work.
What was difficult part is to highlight with word-boundary, skip non-word-char(it's noisy unless doing some filtering).

Why I stopped is
- was not easy as I thought to implement **proper way**(skip unnecessary highlight).
- I'm not sure if I really need this feature by myself(maybe I like it, but priority is low).

I might re-start this feature attempt when I want to, but can't say when.l

**Comment by x0zzz** [comment]
@t9md Alright, thanks for giving it a shot. I hope strong dopamine hit strikes you soon and the overwhelming urge will compel you to finish what you already started :)
Thanks for all your work!
